### ISSUE TITLE
Support keyPrefix option of useTranslation

### ISSUE THREAD

**Issue by incleaf** [i18next/i18next-parser#485]
### Input

```tsx
function Component() {
  const { t } = useTranslation('foo', { keyPrefix: 'bar' })

  t('text1')
  t('text2')
}
```

### Expected output

```js
// foo.json
{
  "bar": {
    "text1": "",
    "text2": ""
  }
}
```

### Actual output

```js
// foo.json
{
  "text1": "",
  "text2": ""
}
```

## Motivation

Currently [`keyPrefix` option](https://react.i18next.com/latest/usetranslation-hook#optional-keyprefix-option) is not supported

**Comment by karellm** [comment]
You can make a PR and I will review it.

**Comment by karellm** [comment]
This is implemented in `6.3.0`
